supabase/migrations/20230924193752_label_changes_rls.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE label_changes ENABLE ROW LEVEL SECURITY
```

Comment:
Whats this for?

Reply:
> Whats this for?

To enable RLS for the DB, more like extra security..
Always enabled on all new tables

Reply:
<img width="660" alt="Screenshot 2023-09-26 at 12 19 31 AM" src="https://github.com/ubiquity/ubiquibot/assets/51956013/d33c5d87-66e4-49a8-90ea-1fc5b1b11f1c">

Reply:
Right but why are we only now adding this?

Reply:
It was added to all tables a while ago, this is just me enabling it because I created a new table